api/src/test/java/org/openmrs/api/EncounterServiceTest.java

```diff
@@ -2528,7 +2531,8 @@ public void getEncounter_shouldFailIfUserIsNotAllowedToViewEncounterByGivenId()
 		// have to add privilege in order to be able to call getEncounter(Integer) method
 		Context.addProxyPrivilege(PrivilegeConstants.GET_ENCOUNTERS);
 		
-		assertNull(Context.getEncounterService().getEncounter(encounter.getId()));
+		//assertNull(Context.getEncounterService().getEncounter(encounter.getId()));
+		assertThrows(APIException.class,() -> assertNull(Context.getEncounterService().getEncounter(encounter.getId())));
```

Comment:
I assume

`Context.getEncounterService().getEncounter(encounter.getId())`

throws an exception. So we do not need to assert that the return value is null since it will not return anything.

You can thus remove the commented line and the call to `assertNull` in the `assertThrows`

Reply:
thanks you @teleivo